webapp/src/app/pacman-features/modules/assets/asset-details/asset-details.component.ts

```diff
@@ -370,7 +370,7 @@ export class AssetDetailsComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   assignCloudType(event:string){
-    const cloudType = event?.toLowerCase();
+    const cloudType = event?.toLowerCase()?.replace(/\s/g, '');
```

Comment:
Why is cloud type a “source” name instead of a source id?

please note that we are moving away of naming things cloud, and instead our plugin=source vs plugin=cloud